cloud-azure/src/main/resources/templates/arm-v2.ftl

```diff
@@ -293,16 +293,17 @@
                    }
                  },
                  {
-                   "apiVersion": "2018-04-01",
+                   "apiVersion": "2019-07-01",
```

Comment:
I guess this does not cause any problem but I need to ask :)

Reply:
Here are the docs for these API versions:
* [2018-04-01](https://docs.microsoft.com/en-us/azure/templates/microsoft.compute/2018-04-01/virtualmachines?tabs=json)
* [2019-07-01](https://docs.microsoft.com/en-us/azure/templates/microsoft.compute/2019-07-01/virtualmachines?tabs=json)

The only incompatible change is this `identity.identityIds` vs `identity.userAssignedIdentities` switch. There were a few additional properties added here and there, but all of them seem optional. In any case, I did not encounter any problems with DL & DH launch.